src/tools/clippy/tests/ui/bind_instead_of_map.stderr

```diff
@@ -2,25 +2,39 @@ error: using `Option.and_then(Some)`, which is a no-op
   --> tests/ui/bind_instead_of_map.rs:8:13
    |
 LL |     let _ = x.and_then(Some);
-   |             ^^^^^^^^^^^^^^^^ help: use the expression directly: `x`
+   |             ^^^^^^^^^^^^^^^^
    |
 note: the lint level is defined here
   --> tests/ui/bind_instead_of_map.rs:1:9
    |
 LL | #![deny(clippy::bind_instead_of_map)]
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+help: use the expression directly
+   |
+LL |     let _ = x;
+   |             ~
 
 error: using `Option.and_then(|x| Some(y))`, which is more succinctly expressed as `map(|x| y)`
   --> tests/ui/bind_instead_of_map.rs:9:13
    |
 LL |     let _ = x.and_then(|o| Some(o + 1));
-   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try: `x.map(|o| o + 1)`
+   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+help: try
```

Comment:
Clippyy needs to have better messages for these suggestions

Reply:
https://rust-lang.zulipchat.com/#narrow/stream/257328-clippy/topic/On.20short.20help.20messages